### PR TITLE
Use uniqueId instead of securityName in CUR

### PR DESCRIPTION
**Pull request by wrodrig**
When using Custom User Registry, provide an option for using the uniqueId instead of securityName.

### CODE REVIEW

**Comment by wrodrig** [comment]
Fixes #21808

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Review by arkarkala — DISMISSED** [review]

**Comment by wrodrig** [comment]
#run-libby-bot

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#run-libby-bot

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_3lGiYD2qEe23scQ7k9ObvQ&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

**Comment by wrodrig** [comment]
#build 
#git.clone.checkout.branch=closed-wlp-changes-21808

**Comment by posmith** [comment]
L2 message review not required

File: dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -26,7 +26,10 @@ allowHashtableLoginWithIdOnly=Allow hashtable login with ID only
...
-fillInGroupInformationForHashtableLogin.desc=Specifies whether the group information for the login user should be automatically populated when performing a Hashtable login. If true and no groups are provided to the Hashtable login, then the group information will be populated. If true and group information is provided, then the provided group information is used. If false, then the group information, if provided, is used. 
+fillInGroupInformationForHashtableLogin.desc=Specifies whether the group information for the login user should be automatically populated when performing a Hashtable login. If true and no groups are provided to the Hashtable login, then the group information will be populated. If true and group information is provided, then the provided group information is used. If false, then the group information, if provided, is used.
+
+useDisplayNameForSecurityName=Use displayName as securityName in customUR
+useDisplayNameForSecurityName.desc=Specifies if the displayName should be returned for programmatic APIs like getRemoteUser, getCallerPrincipal, getUserPrincipal when using Custom User Registry.
```
Comment (helyarp) [inline dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties]: To remove "should be," replace "Specifies if the displayName should be returned" with "Specifies whether to return  displayName"

**Review by helyarp — COMMENTED** [review]
Suggested change

**Comment by wrodrig** [comment]
#build

File: dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (wrodrig) [reply dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties]: Thanks I changed it

**Comment by wrodrig** [comment]
#build

**Review by helyarp — COMMENTED** [review]
Changes look good!

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#run-libby-bot

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#build

**Comment by wrodrig** [comment]
#run-libby-bot

**Review by arkarkala — APPROVED** [review]

**Pull request merged**